src/styles/stopwatch.scss

```diff
@@ -0,0 +1,59 @@
+.stopwatch {
+  position: relative;
+  width: 80vmin;
+  height: 80vmin;
+  border: $border-width dotted $border-color;
+  border-radius: 50%;
+
+  &__minutes-hand {
+    position: absolute;
+    height: $minutes-hand-height;
+    width: $hand-width-minutes;
+    background-color: $minutes-hand-color;
+    bottom: 50%;
+    left: calc(50% - (#{$hand-width-minutes} / 2));
+    transform-origin: bottom center;
+    animation: rotate 3600s steps(60) infinite;
```

Comment:
The `animation` shorthand is used correctly, but for better readability and maintenance, consider breaking it into individual properties. This way, it's clearer to see and manage each aspect of the animation (e.g., `animation-name`, `animation-duration`, `animation-timing-function`, etc.).